app/Components/Statistics/TotalStatistics.tsx

```diff
@@ -20,6 +27,27 @@ const TotalStatistics = (props: TotalStatisticsProps) => {
 
   const theme = useTheme();
 
+  // sort by most number of hints
+  const numHintsUsedPerStrategy = [...props.numHintsUsedPerStrategy].sort(
+    (a, b) => {
+      return b.numHintsUsed - a.numHintsUsed;
+    }
+  );
+
+  const strategyHints: React.JSX.Element[] = [];
```

Comment:
nit: a single line comment summarizing what the following code does would be nice (my reasoning is code is written once and read many times so saving a little bit of time understanding it in each future read is worth it)

Reply:
I pushed up this change